src/pip/_internal/operations/prepare.py

```diff
@@ -140,8 +140,7 @@ def prepare_linked_requirement(
             # inconsistencies are logged later, but do not fail the
             # installation.
             # FIXME: this won't upgrade when there's an existing
-            # package unpacked in `req.source_dir`
-            # package unpacked in `req.source_dir`
+            #  package unpacked in `req.source_dir`
```

Comment:
```suggestion
            # package unpacked in `req.source_dir`
```

Reply:
PyCharm and (IIRC) VSCode will extend TODO, XXX, and FIXME highlighting to subsequent lines if they are indented by 1 or more spaces.